internal/config/config_test.go

```diff
@@ -753,17 +774,21 @@ func TestParse(t *testing.T) {
 		{
 			desc: "bad community literal (wrong format)",
 			crs: ClusterResources{
-				Pools: []v1beta1.AddressPool{
+				Pools: []v1beta1.IPPool{
 					{
 						ObjectMeta: v1.ObjectMeta{Name: "pool1"},
-						Spec: v1beta1.AddressPoolSpec{
```

Comment:
You can delete the Pools assignment in this test as it is not holding the community definition anymore.
also in the next 2 community tests below it.

Reply:
Done